lib/rust/ensogl/component/scroll-area/src/lib.rs

```diff
@@ -101,11 +119,20 @@ impl ScrollArea {
     pub fn new(app: &Application) -> ScrollArea {
         let scene = &app.display.default_scene;
         let logger = Logger::new("ScrollArea");
+        let camera = scene.layers.main.camera();
         let display_object = display::object::Instance::new(&logger);
+        let masked_layer = MaskedLayer::new(&logger, &camera);
+        let display_object = display::object::InstanceWithLayer::new(display_object, masked_layer);
 
         let content = display::object::Instance::new(&logger);
         display_object.add_child(&content);
 
+        let mask = mask::View::new(&logger);
+        display_object.add_child(&mask);
+
+        display_object.layer.masked_layer.add_exclusive(&content);
+        display_object.layer.mask_layer.add_exclusive(&mask);
```

Comment:
Do I understand correctly that you are adding background of the scroll area to the masked layer here? If so, this is wrong, because the background uses a shape with rounded corners, just as a mask. This means that rounded corners alpha of mask and background will be multiplied causing it to be pixelated without nice anti-aliasing.

Reply:
Not, the scroll area itself does not have a background. A background in the demo scene is a separate shape, not attached to the scroll area at all. This doesn't seem correct and I will probably attach the background to the scroll area display object, but it will live in a separate scene layer anyway. `masked_layer` is used only for area's content, nothing more.

Reply:
Add info to the doc of `ScrollArea` that it doesnt contain background, please